integration-tests/sandbox-ballot-operation.spec.ts

```diff
@@ -1,8 +1,8 @@
 import { CONFIGS, sleep } from './config';
 
-CONFIGS().forEach(async ({ lib, protocol, setup, rpc }) => {
+CONFIGS().forEach(async ({ lib, protocol, setup }) => {
   const Tezos = lib;
-  const flextesanet = (rpc === 'http://0.0.0.0:20000') ? it : it.skip;
+  // const flextesanet = rpc === 'http://localhost:20000' ? it : it.skip;
```

Comment:
can remove?

Reply:
we can, but it still valid since it's going to be reinstated in the very near future.